vault/resource_aws_auth_backend_client.go

```diff
@@ -69,6 +73,11 @@ func awsAuthBackendClientResource() *schema.Resource {
 				Optional:    true,
 				Description: "Region to override the default region for making AWS STS API calls.",
 			},
+			useSTSRegionFromClient: {
```

Comment:
Should this be marked as Computed?

Reply:
+1 on adding Computed to support the default :)